<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentMainWindow</class>
 <widget class="QMainWindow" name="StudentMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1636</width>
    <height>911</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Personal Information</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {background-color: rgb(235, 239, 240);}
QPushButton {background-color: rgb(0, 0, 127); color: white; border: 2 px; border-radius: 10 px;}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="studentView">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>1281</width>
      <height>731</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>191</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>301</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Student</string>
    </property>
   </widget>
   <widget class="QPushButton" name="classButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>249</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>301</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Class</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="gradeButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>307</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>301</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Grades</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1636</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
